### DISCUSSION TITLE
Plugins from MR Software GmbH discontinued?

### DISCUSSION THREAD

**Discussion by klausjettkant** [kimai/kimai#4240] [General] [answered]
Hi,

i am using the plugins "Controlling", "Sick-Day" and "Vacation" build by MR Software GmbH. The company shut down in june 2023, there is no plugin-support for Kimai 2.0. Does anyone know if these or similar plugins will be available for future Kimai versions?

Thank you in advance,
Klaus Jettkant

**Comment by kevinpapst** [comment, marked as answer]
See https://github.com/kimai/kimai/discussions/4145
